support-frontend/app/controllers/Application.scala

```diff
@@ -65,8 +65,8 @@ class Application(
   def geoRedirect: Action[AnyContent] = GeoTargetedCachedAction() { implicit request =>
     val redirectUrl = request.geoData.countryGroup match {
       case Some(UK) => buildCanonicalShowcaseLink("uk")
-      case Some(US) => "/us/contribute"
-      case Some(Australia) => "/au/contribute"
+      case Some(US) => "/us/support"
```

Comment:
This seems unrelated to this change and will conflict with the PR to do this work, I'd revert it